kimchi/src/circuits/domain_constant_evaluation.rs

```diff
@@ -0,0 +1,79 @@
+use crate::circuits::domains::EvaluationDomains;
```

Comment:
would be nice to add a module-level comment (with `//!`)

Reply:
added something but im not confident at my description. could you suggest something better?

Reply:
maybe "This contains the [DomainConstantEvaluations] which is used to provide precomputations to a [ConstraintSystem]."